clip_retrieval/clip_back.py

```diff
@@ -435,8 +435,8 @@ def post(self):
         text_input = json_data.get("text", None)
         image_input = json_data.get("image", None)
         image_url_input = json_data.get("image_url", None)
-        modality = json_data["modality"]
-        num_images = json_data["num_images"]
+        modality = json_data.get("modality", "image")
```

Comment:
Having some default would be convenient. I am not sure if defaults aren't set intentionally, feel free to let me know.

Reply:
yes I think it's better if it's explicitely sent by the front / caller